src/model_signing/_cli.py

```diff
@@ -255,21 +255,32 @@ def _sign_private_key(
     signature.
 
     Traditionally, signing could be achieved by using a public/private key pair.
-    Pass the signing key using `--private_key`.
+    Pass the signing key using `--private_key`. The private key may also be
+    a PKCS #11 URI.
 
     Note that this method does not provide a way to tie to the identity of the
     signer, outside of pairing the keys. Also note that we don't offer key
     management protocols.
     """
     try:
-        model_signing.signing.Config().use_elliptic_key_signer(
-            private_key=private_key, password=password
-        ).set_hashing_config(
-            model_signing.hashing.Config().set_ignored_paths(
-                paths=list(ignore_paths) + [signature],
-                ignore_git_paths=ignore_git_paths,
-            )
-        ).sign(model_path, signature)
+        if private_key.startswith("pkcs11:"):
```

Comment:
I'm a little bit concerned by this. all other arguments are real paths, and we can use `click` features to test when these files exists or display properly formatted error messages. We also want to add support for [cloud filesystems](https://github.com/sigstore/model-transparency/issues/148), so now the argument would have 3 separate meanings (local path, cloud path, pkcs alternative).

Can we actually separate this?